ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,314 @@
+Liquibase command 'update' was executed successfully.
+```
+
+The result will be deleting the index, adding the `is_deleted` column, disabling the auto partitioning setting, and creating a topic:
+
+![../_assets/liquibase-step-2.png](../_assets/liquibase-step-2.png)
+
+### Initializing liquibase in a project with a non-empty data schema
+
+Suppose I have an existing project with the current database schema:
+
+![../_assets/liquibase-step-3.png](../_assets/liquibase-step-3.png)
+
+To start using liquibase, you need to run `liquibase generate-changelog --changelog-file=changelog.xml`.
+
+The content of the generated changelog.xml:
+
+```xml
+<changeSet author="kurdyukov-kir (generated)" id="1711556283305-1">
+    <createTable tableName="all_types_table">
+        <column name="id" type="INT32">
+            <constraints nullable="false" primaryKey="true"/>
+        </column>
+        <column name="bool_column" type="BOOL"/>
+        <column name="bigint_column" type="INT64"/>
+        <column name="smallint_column" type="INT16"/>
+        <column name="tinyint_column" type="INT8"/>
+        <column name="float_column" type="FLOAT"/>
+        <column name="double_column" type="DOUBLE"/>
+        <column name="decimal_column" type="DECIMAL(22, 9)"/>
+        <column name="uint8_column" type="UINT8"/>
+        <column name="uint16_column" type="UINT16"/>
+        <column name="unit32_column" type="UINT32"/>
+        <column name="unit64_column" type="UINT64"/>
+        <column name="text_column" type="TEXT"/>
+        <column name="binary_column" type="BYTES"/>
+        <column name="json_column" type="JSON"/>
+        <column name="jsondocument_column" type="JSONDOCUMENT"/>
+        <column name="date_column" type="DATE"/>
+        <column name="datetime_column" type="DATETIME"/>
+        <column name="timestamp_column" type="TIMESTAMP"/>
+        <column name="interval_column" type="INTERVAL"/>
+    </createTable>
+</changeSet>
+<changeSet author="kurdyukov-kir (generated)" id="1711556283305-2">
+    <createTable tableName="episodes">
+        <column name="series_id" type="INT64">
+            <constraints nullable="false" primaryKey="true"/>
+        </column>
+        <column name="season_id" type="INT64">
+            <constraints nullable="false" primaryKey="true"/>
+        </column>
+        <column name="episode_id" type="INT64">
+            <constraints nullable="false" primaryKey="true"/>
+        </column>
+        <column name="title" type="TEXT"/>
+        <column name="air_date" type="DATE"/>
+    </createTable>
+</changeSet>
+<changeSet author="kurdyukov-kir (generated)" id="1711556283305-3">
+    <createIndex indexName="title_index" tableName="episodes">
+        <column name="title"/>
+    </createIndex>
+</changeSet>
+```
+
+Then you need to synchronize the generated changelog.xml the file, this is done by the command `liquibase changelog-sync --changelog-file=dbchangelog.xml`.
```

Comment:
It's not clear what this is supposed to do: why the filename is different? We now have two changelog files? If so, what's the difference between them?